app/assets/javascripts/views/form_view.js

```diff
@@ -116,13 +116,13 @@ module.exports = Backbone.View.extend({
     return el;
   },
 
-  bindElementToAttribute: function(el, name, eventType) {
+  bindElementToAttribute: function(el, name, eventType, silent = true) {
```

Comment:
What do you think about adding a options object instead of a new parameter? That way we avoid to adding a new parameter every time we need a new option here.

You can do something like:

```js
bindElementToAttribute: function(el, name, eventType, { silent = true } = {}) {
 // ...
```